daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -380,7 +386,12 @@ class Runner(compiledPackages: CompiledPackages, script: Script.Action, timeMode
                       client <- Converter.toFuture(
                         clients
                           .getPartyParticipant(Party(party.value)))
-                      submitRes <- client.submit(party, commands)
+                      commitLocation <- if (vals.size == 3) {
```

Comment:
maybe worth adding a comment that this is for backwards compat.

Reply:
done